pom.xml

```diff
@@ -37,12 +37,9 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-
         <dependency>
-            <groupId>org.gwtopenmaps.openlayers</groupId>
-            <artifactId>gwt-openlayers-showcase</artifactId>
-            <version>1.1.0</version>
-            <type>war</type>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-webflux</artifactId>
```

Comment:
I would recommend to avoid webflux and reactive things right now. Spring MVC should be enough for this, firstly you need to know and understand Spring MVC instead of webflux and reactive things.